force-app/main/default/classes/AccountModelSettingsController_TEST.cls

```diff
@@ -0,0 +1,146 @@
+            Test.stopTest();
+        } catch (AuraHandledException e) {
+            System.assertEquals('Script-thrown exception', e.getMessage(), 'Should throw exception.');
+        }
+    }
+
+    /**************************************************************************************************************************
+     ****************************************************** STUBS **************************************************************
+     **************************************************************************************************************************/
+
+    /**************************************************************************************************************************************
+     * @description Stub class to simulate the response from AccountModelSettingsVMapper to return a valid AccountModelSettingsVModel
+     **************************************************************************************************************************************/
+    private class STUB_AccountModelSettingsVMapper extends AccountModelSettingsVMapper {
+        public override AccountModelSettingsVModel getAccountModelSettingsViewModel() {
+            List<ComboboxValueVModel> accountRecordTypeOptionsList = new List<ComboboxValueVModel>();
+
+            for (Integer i = 0; i < loopCounter; i++) {
+                accountRecordTypeOptionsList.add(new ComboboxValueVModel('Label' + i, 'Value' + i));
+            }
+
+            ComboboxVModel defaultAccountComboboxVModel = new ComboboxVModel(
+                'Selected Default Account',
+                accountRecordTypeOptionsList
+            );
+            ComboboxVModel hhAccountComboboxVModel = new ComboboxVModel(
+                'Selected HH Account',
+                accountRecordTypeOptionsList
+            );
+            ComboboxVModel adminAccountComboboxVModel = new ComboboxVModel(
+                'Selected Admin Account',
+                accountRecordTypeOptionsList
+            );
+
+            return new AccountModelSettingsVModel(
+                defaultAccountComboboxVModel,
+                hhAccountComboboxVModel,
+                adminAccountComboboxVModel
+            );
+        }
+    }
+
+    /**************************************************************************************************************************************
+     * @description Stub class to simulate the response from AccountModelSettingsVMapper to return a valid AccountModelSettingsVModel
+     **************************************************************************************************************************************/
+    private class STUB_ExceptionAccountModelSettingsVMapper extends AccountModelSettingsVMapper {
+        public override AccountModelSettingsVModel getAccountModelSettingsViewModel() {
+            throw new TestException('Test exception!');
```

Comment:
You can always use a standard exception like a NoAccessException, but that does run the risk of us catching it later. This may be a better solution than doing so.

This is a perfectly-functional piece of code and does not require changes.